keras_resnet/blocks/_1d.py

```diff
@@ -29,18 +25,48 @@ def basic_1d(filters, stage=0, block=0, kernel_size=3, numerical_name=False, str
 
     :param kernel_size: size of the kernel
 
-    :param numerical_name: if true, uses numbers to represent blocks instead of chars (ResNet{101, 152, 200})
```

Comment:
Why remove the numerical_names? Did you find another way to fix this?

Reply:
It’s still a WIP, but I want to standardize names (using the names from KaimingHe/deep-residual-networks) since I’m going to start uploading a suite of weights.

Reply:
iirc the issue was that the names of KaimingHe weren't really following a straight set of rules. In one network he uses `a/b/c/...` and in the next he uses `a/b1/b2/b3/b4/...`. How do you plan to handle this?